Source/Demos/Demo.PrimitiveBatch/Game1.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using MonoGame.Extended.Graphics;
+using MonoGame.Extended.Graphics.Batching;
+using SpriteBatch = MonoGame.Extended.Graphics.Batching.SpriteBatch;
```

Comment:
Hmm.. I don't think it's a good idea to call this new thing a `SpriteBatch`. It's going to get very confusing when it only differs by namespace. I'd rather prefix or suffix the name with another word to make it really clear you're using an alternative to the normal sprite batch.